docs/source/developer_guide/architecture.md

```diff
@@ -45,32 +45,15 @@ The organization of Morpheus can be broken down into four different layers. Work
 
 ## Pipeline Details
 
-Pipelines are a collection of one or more stages that are connected via edges. Data flows from one stage to the next
-across these edges using buffers. We utilize these buffers to allow stages to process messages at different rates. Once
-each stage is done processing a message, the pipeline will move it onto the next stage's buffer for processing. This
-process continues until the message has made it through the entire pipeline.
+Pipelines are a collection of one or more stages that are connected via edges. Data flows from one stage to the next across these edges using buffers. We utilize these buffers to allow stages to process messages at different rates. Once each stage is done processing a message, the pipeline will move it onto the next stage's buffer for processing. This process continues until the message has made it through the entire pipeline.
```

Comment:
nit: via can be avoided with "by using," "using," or "by".  In this, "along" might work too.

nit: s/utilize/use/.  How did this not get flagged by the Microsoft [Wordiness](https://github.com/errata-ai/Microsoft/blob/c798f131106d77c136a3c1b5fa4916dee1de8edd/Microsoft/Wordiness.yml#L13) rule?  Mind you, I don't think we'll eventually adopt all the Microsoft rules--but this rule or some variation will survive.

Reply:
I believe this isn't flagged as we set `MinAlertLevel = error` without this, it would have flagged way too much stuff that would have been too far outside the scope of this PR, where I'm really just trying to get this in place for CI and fix any obvious spelling mistakes.